docs/docsite/rst/dev_guide/developing_collections_creating.rst

```diff
@@ -17,6 +17,33 @@ A user can then install your collection on their systems.
    :local:
    :depth: 2
 
+Naming your collection
+======================
+
+Collection names consist of a namespace and a name, separated by a period (``.``). Both namespace and name should be valid Python identifiers. This means that they should consist of ASCII letters, digits, and underscores.
+
+.. note::
+
+    Usually namespaces and names use lower-case letters, digits, and underscores, but no upper-case letters.
+
+You should make sure that the namespace you use is not registered by someone else by checking on `Ansible Galaxy's namespace list <https://galaxy.ansible.com/ui/namespaces/>`__. If you chose a namespace or even a full collection name that collides with another collection on Galaxy, it can happen that if you or someone else runs ``ansible-galaxy collection install`` with your collection name, you end up with another collection. Even if the namespace currently does not exist, it could be created later by someone else.
+
+If you want to request a new namespace on Ansible Galaxy, `create an issue on github.com/ansible/galaxy <https://github.com/ansible/galaxy/issues/new?assignees=thedoubl3j%2C+alisonlhart%2C+chynasan%2C+traytorous&labels=area%2Fnamespace&projects=&template=New_namespace.md&title=namespace%3A+FIXME>`__.
+
+There are a few special namespaces:
+
+:ansible:
+
+  The `ansible` namespace <https://galaxy.ansible.com/ui/namespaces/ansible/>`__ is owned by Red Hat and reserved for official Ansible collections. Two special members are the synthetic ``ansible.builtin`` and ``ansible.legacy`` collections. These cannot be found on Ansible Galaxy, but are built-in into ansible-core.
```

Comment:
`ansible` needs double backticks and the link is misformatted:

![image](https://github.com/ansible/ansible-documentation/assets/9920591/208c57ce-f2bb-4681-b460-8748e82ee5de)

Reply:
Formatting inside link titles doesn't really work with RST, so it will be without double backticks. There was an additional single backtick inside the link title which screwed up formatting...
```suggestion
  The `ansible namespace <https://galaxy.ansible.com/ui/namespaces/ansible/>`__ is owned by Red Hat and reserved for official Ansible collections. Two special members are the synthetic ``ansible.builtin`` and ``ansible.legacy`` collections. These cannot be found on Ansible Galaxy, but are built-in into ansible-core.
```